app/models/solr_document.rb

```diff
@@ -35,6 +35,19 @@ def embeddable?(blacklight_config = CatalogController.blacklight_config)
     false
   end
 
+  def iiifable?
+    iiif_manifest_url.present?
+  end
+
+  def iiif_manifest_url
+    case first('agg_provider.en_ssim')
+    when 'Bodleian Libraries'
+      "https://iiif.bodleian.ox.ac.uk/iiif/manifest/#{first('agg_is_shown_at_ssim')&.split('/')&.last}.json"
+    else
```

Comment:
Should we just omit the whole `else` block? But, 🤷‍♂ , hopefully this whole method isn't long for this world.